src/scenes/ApplicationRouter/provider/index.tsx

```diff
@@ -72,10 +81,25 @@ const ApplicationProvider = ({ children }: Props) => {
 
   useEffect(() => {
     if (selectedApplication) {
-      const dispatched = dispatch(requestListApplicationModules({ applicationId: selectedApplication.id }));
-      setListApplicationModulesRequest(dispatched.requestId);
+      const listModulesDispatched = dispatch(requestListApplicationModules({ applicationId: selectedApplication.id }));
+      setListApplicationModulesRequest(listModulesDispatched.requestId);
+
+      const listDeliverablesDispatched = dispatch(
+        requestListApplicationDeliverables({ applicationId: selectedApplication.id })
+      );
+      setListApplicationDeliverablesRequest(listDeliverablesDispatched.requestId);
     }
-  }, [dispatch, selectedApplication, setListApplicationModulesRequest]);
+  }, [dispatch, selectedApplication, setListApplicationModulesRequest, setListApplicationDeliverablesRequest]);
+
+  useEffect(() => {
+    if (
+      listApplicationDeliverablesResult &&
+      listApplicationDeliverablesResult.status === 'success' &&
+      listApplicationDeliverablesResult.data
+    ) {
```

Comment:
This can be shortened to:

```suggestion
    if (
      listApplicationDeliverablesResult?.data &&
      listApplicationDeliverablesResult.status === 'success'
    ) {
```